packages/input/tests/__snapshots__/input.test.tsx.snap

```diff
@@ -16,8 +16,6 @@ exports[`Elements inside input render correctly 1`] = `
   width: 2.5rem;
   height: 2.5rem;
   font-size: 1rem;
-  padding-left: 1rem;
```

Comment:
I am not quite sure about these changes.
I just changed the code from using `pl` and `pr` to `px`.